### ISSUE TITLE
RandomForest Not Working with Certain Datasets

### ISSUE THREAD

**Issue by washcycle** [edgararuiz/tidypredict#18]
Using a RandomForest causes an error from the purr package with certain datasets.

It was on one of the RStudio Conference 2018 "Big Data in R" EC2 machines.

```
library(randomForest)
model <- randomForest(mpg ~ ., data = mtcars)
tidypredict_sql(model, con) # any connection I tried caused the same error
```

**ERROR**
`Error: `.x` is empty, and no `.init` supplied` 
which comes from the purr package.

**Comment by washcycle** [comment]
I tracted it down to right here so far

`parsemodel.R`

`  all_paths <- model_frame %>%
    filter(.data$status == -1) %>%
    pull(.data$rowid) %>%
    map(~get_path(.x, model_frame)) %>%
    bind_rows()`

all_paths comes back empty

**Comment by washcycle** [comment]
When fitting a random forest for regression the status variables doesn't contain 1. It contains -1 and -3 that might be the issue. Can use `randomForest::getTrees(...)` on the model for a regression and classification case to see what I am talking about.

**Comment by washcycle** [comment]
I think I figured out the issue. The `randomForest::getTrees` has a different criteria for regression problems.

> For numerical predictors, data with values of the variable less than or equal to the splitting point go to the left daughter node.

**Comment by edgararuiz** [comment]
Fixed with #19 - Thanks @washcycle!
